update/upgrade_db.sh

```diff
@@ -28,25 +36,25 @@ case $key in
     -h|--help)
         echo "Arguments:"
         echo -e "\t-c|--clean\t\tCleans comp and test DB before starting"
-        # echo -e "\t-d|--localdirpath\tRuns a second update cycle with local customization delta files"
+        echo -e "\t-d|--localdirpath\tRuns a second update cycle with local customization delta files"
```

Comment:
In my opinion there should not be a second update cycle. The core, extensions and customizations should be updated as part of the same pum upgrade cycle.